components/zoho_crm/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@pipedream/zoho_crm",
-    "version": "0.0.1",
+    "version": "0.3.4",
```

Comment:
Is this the correct version?

Reply:
It is. As far as I know it has to do with the lerna version [here](https://github.com/PipedreamHQ/pipedream/blob/master/lerna.json#L7)